src/components/Book.jsx

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => (
-  <tr>
-    <td>{book.id}</td>
-    <td>{book.title}</td>
-    <td>{book.category}</td>
-  </tr>
-);
+const Book = ({ book, removeBook }) => {
+  const handleRemove = () => removeBook(book);
```

Comment:
This should be implemented in the `BookList` component. Then pass the `handleRemove` to `removeBook` props of the `Book` component.
```suggestion
```